server/auth/index.js

```diff
@@ -0,0 +1,25 @@
+const jwt = require('jsonwebtoken');
```

Comment:
Middleware to add to protected routes.
Checks to see that a valid json web token has been provided with the request.